### ISSUE TITLE
Export bounces to CSV

### ISSUE THREAD

**Issue by benstaf** [knadh/listmonk#1895]
I want to export bounces to CSV file 

Now I only can export all subscribers, there's no such export button for bounces

that's a problem for large databases

![Screenshot_2024-06-13-20-08-42-341_com android chrome-edit](https://github.com/knadh/listmonk/assets/8183572/0cc5f894-7bf9-4afc-8990-1ec66e2543d4)
![Screenshot_2024-06-13-20-08-23-346_com android chrome-edit](https://github.com/knadh/listmonk/assets/8183572/abfaad24-0963-4180-aaff-5f7c4b9e6d70)

**Comment by benstaf** [comment]
It's possible to query the database with "Advanced"

**Comment by CodeAlDente** [comment]
I just ran into this as well. A button to export bounces would be nice, but it could include duplicates and entries not associated with a specific campaign, so I had to filter those out afterward. Using the advanced query box is much easier in this case.

To get all bounces, simply use the following:

```sql
EXISTS (
  SELECT 1
  FROM bounces
  WHERE bounces.subscriber_id = subscribers.id
)
```

I think it would make sense to have a dedicated page where a collection of common queries like this could be listed.
